content/courses/tokens-and-nfts/nfts-with-metaplex.md

```diff
@@ -575,49 +655,59 @@ following:
 
 ```
 % npx esrun create-metaplex-nft.ts
-Loaded user: 4DRi8LxWhAWf9LwUuyb3Rz4Y4USEKzjdvQBWTyrkJtwg
-image uri: https://arweave.net/Z2JoDr7W4A3mWE9aTq8ex13IoPx1v1QewjVqPKPItuE
-Token Mint: https://explorer.solana.com/address/HCq8ERk1PSLCuBJeWx7du4dom3YbyfKpaCWDQ3Eft6aM?cluster=devnet
-Created NFT address is HCq8ERk1PSLCuBJeWx7du4dom3YbyfKpaCWDQ3Eft6aM
+
+Loaded user: 4kg8oh3jdNtn7j2wcS7TrUua31AgbLzDVkBZgTAe44aF
+image uri: https://arweave.net/nVjCW_g-UMQUZano8vh-xPLX4nHwd5LsC8hXEuMdk_w
+NFT offchain metadata URI: https://arweave.net/N2HivyLiwRdb4CutMSM9aDXMSMyS0PF_VSuAfKGUhhI
+Token Mint:  https://explorer.solana.com/address/Zxd9TmtBHQNti6tJxtx1AKYJFykNUwJL4rth441CjRd?cluster=devnet
+Created NFT address is Zxd9TmtBHQNti6tJxtx1AKYJFykNUwJL4rth441CjRd
 ✅ Finished successfully!
 ```
 
 Inspect your NFT at the address given! If you have any trouble, try and fix it
 yourself, but if you need to you can also check out the
-[solution code](https://github.com/solana-developers/professional-education/blob/main/labs/create-metaplex-nft.ts).
+[solution code](https://github.com/solana-developers/professional-education/blob/main/labs/metaplex-umi/create-nft.ts).
```

Comment:
Add an image here of what Explorer should look like. See CONTRIBUTING for image specifications.